### ISSUE TITLE
gixpp: add `OCCURS UNBOUNDED`

### ISSUE THREAD

**Issue by GitMensch** [mridoni/gixsql#12]
```cobol
       01 L-DYNBUFFER.
          05 FILLER     OCCURS UNBOUNDED
                        DEPENDING ON L-DYNBUFFER-LEN
                        PIC X.
```

currently leading to

> syntax error, unexpected WORD, expecting NUMERIC

Note: not necessary for my personally as the workaround in gixsql of putting that into `EXEC SQL IGNORE` works.

**Comment by mridoni** [comment]
Fixed in [v1.0.15](https://github.com/mridoni/gixsql/releases/tag/v1.0.15)

**Comment by GitMensch** [comment]
> GnuCOBOL emits a few `warning: OCCURS DEPENDING ON without TO phrase used [-Wdialect]` but I guess that is to be expected

Yes, for all `OCCURS ... DEPENDING ON` the standard requests a "from" value; all compilers I know of that support it without the from assume 1 in this case, so does GnuCOBOL. To be standard conforming and _sure_ that it works identical on all COBOL environments and to be actually more "clear" it is better to explicit state ´0 TO` or `1 TO` (or whatever the minimum for the code is.
